merlin/systems/dag/ops/session_filter.py

```diff
@@ -17,21 +17,52 @@
 import json
 
 import numpy as np
-
 from merlin.dag import ColumnSelector, Node
 from merlin.schema import ColumnSchema, Schema
+
 from merlin.systems.dag.ops.operator import InferenceDataFrame, PipelineableInferenceOperator
 
 
 class FilterCandidates(PipelineableInferenceOperator):
-    def __init__(self, filter_out, input_col=None):
+    """
+    This operator takes the input column and filters out elements of that column
```

Comment:
The "out" seems technically unnecessary, but it's OK as-is.  Could be "...and removes elements...". The description is perfectly clear.  Nice.